server.js

```diff
@@ -23,6 +41,37 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  res.json(thoughts)
+})
+
+app.post('/thoughts', async (req, res) => {
+  // retrive info sent by client
+  const { message } = req.body;
+  // use mongoose model to create the database entry
+  const thought = new Thought({ message })
+
+  try {
+    //success
+    const savedThought = await thought.save()
+    res.status(201).json(savedThought)
+  } catch (err) {
+    res.status(400).json({ message: 'could not save thought to the database', error: err.errors })
+  }
+})
```

Comment:
according to documentation `post "/"` need to create a new one.

Reply:
Not sure I understand this either?